package.json

```diff
@@ -4,60 +4,65 @@
   "license": "MIT",
   "angular-cli": {},
   "scripts": {
+    "ng": "ng",
     "start": "yarn build-themes && ng serve",
     "start-jit": "yarn build-themes && ng serve --aot=false",
-    "lint": "tslint -p src/tsconfig.json",
+    "start:prod": "yarn build-themes && ng serve --prod",
+    "lint": "tslint -p tsconfig.json",
     "test": "yarn build-themes && ng test",
     "pree2e": "webdriver-manager update",
-    "e2e": "protractor",
+    "e2e": "yarn build-themes && ng e2e",
+    "build": "yarn build-themes && ng build",
+    "build:highmem": "yarn build-themes && node --max_old_space_size=8192 ./node_modules/@angular/cli/bin/ng build",
+    "build:sm": "ng build --prod --source-map",
     "build-themes": "bash ./tools/build-themes.sh",
-    "prod-build": "yarn build-themes && ng build --aot --prod",
+    "prod-build": "yarn build-themes && ng build --prod",
     "postinstall": "webdriver-manager update --gecko false && yarn build-themes",
     "publish-prod": "bash ./tools/deploy.sh stable prod",
     "publish-dev": "bash ./tools/deploy.sh",
     "publish-beta": "bash ./tools/deploy.sh stable beta"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^8.0.0",
-    "@angular/cdk": "8.2.1",
-    "@angular/cdk-experimental": "8.2.1",
-    "@angular/common": "^8.0.0",
-    "@angular/compiler": "^8.0.0",
-    "@angular/core": "^8.0.0",
-    "@angular/forms": "^8.0.0",
-    "@angular/material": "8.2.1",
-    "@angular/material-examples": "angular/material2-docs-content#8.2.x",
-    "@angular/material-experimental": "8.2.1",
-    "@angular/material-moment-adapter": "8.2.1",
-    "@angular/platform-browser": "^8.0.0",
-    "@angular/platform-browser-dynamic": "^8.0.0",
-    "@angular/router": "^8.0.0",
-    "core-js": "^2.6.1",
+    "@angular/animations": "^8.2.8",
+    "@angular/cdk": "^8.2.2",
+    "@angular/cdk-experimental": "^8.2.2",
+    "@angular/common": "^8.2.8",
+    "@angular/compiler": "^8.2.8",
+    "@angular/core": "^8.2.8",
+    "@angular/forms": "^8.2.8",
+    "@angular/material": "^8.2.2",
+    "@angular/material-examples": "angular/material2-docs-content#518c443",
```

Comment:
Could this be `#master` until we release v9?

Reply:
This commit sha maps to the latest `8.2.x` snapshot release prior to `8.2.2-0462599` which yarn won't install due to invalid semver. Reported in: https://github.com/angular/components/issues/17264 and fixed in PR https://github.com/angular/components/pull/17267.

Just leaving it as `angular/material2-docs-content` is equivalent to `#master` AFAIK and that does indeed build just fine.

I'm not sure whether or not all of the docs changes for `8.2.x` are getting merged down to `master` though?

Reply:
We merge everything into master and then cherry-pick into `8.2.x` (except odd/exceptional cases)